src/codeu/chat/ServerMain.java

```diff
@@ -33,6 +35,8 @@
 final class ServerMain {
 
   private static final Logger.Log LOG = Logger.newLog(ServerMain.class);
+  
+  private static int myPort;
```

Comment:
Do these need to be static class variables? Can this just be in main? I don't think I see it being used anywhere else other than main, but I may be wrong.